lib/tasks/export_copy_card_orders.rake

```diff
@@ -7,7 +7,7 @@ namespace :reports do
 
       # Run the report for one week up to end of day on the most recent Sunday
```

Comment:
Does this comment need updating? Or was it just wrong before?

Reply:
Yes, it needs to be updated.